pkg/rbac/rbac_test.go

```diff
@@ -28,18 +28,18 @@ func TestRBAC_Permissions(t *testing.T) {
 	}{
 		{AuditRead, 2},
 		{APIKeyRead, 4},
-		{APIKeyWrite, 8},
+		{APIKeyWrite, 12},
```

Comment:
This will technically break anyone using old permissions, but this code has never been in a release.